pkg/test/framework/components/echo/kube/deployment.go

```diff
@@ -125,8 +125,8 @@ spec:
           httpGet:
             path: /
             port: 8080
-          initialDelaySeconds: 10
-          periodSeconds: 10
+          initialDelaySeconds: 1
```

Comment:
echo takes a long time to get ready because of this. Bring it down to lower levels. Note: this is the exact same as istio-proxy settings